compiler/rustc_const_eval/src/interpret/eval_context.rs

```diff
@@ -616,19 +616,11 @@ impl<'mir, 'tcx: 'mir, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
                     match self.size_and_align_of(metadata, &field)? {
                         Some(size_and_align) => size_and_align,
                         None => {
-                            // A field with extern type.  If this field is at offset 0, we behave
-                            // like the underlying extern type.
+                            // A field with extern type. We behave like the underlying type.
```

Comment:
This comment certainly does not apply any more -- this referred to the fact that newtypes around extern type should work, but now with non-ZST fields that is not enough any more.